README.md

```diff
@@ -210,3 +211,10 @@ same message could be an error *or* a warning for example.
 
 Lastly when the processing is complete the validator will output
 the collected data as text or JSON.
+
+## Deploys
+
+We deploy to npm automatically using TravisCI. To release a new version,
+increment the version in `package.json` and create a PR. After the PR is
+merged create a new release with the same tag name as your new version.
```

Comment:
Could you add a link to the releases page on the project?

You could also make a note that the release title and tag should be a version e.g. `0.0.7`